### PR TITLE
Check rosdoc_lite for errors in CI

### PR DESCRIPTION
**Pull request by agutenkunst**
## Description

Needs 

- [x]  https://github.com/PilzDE/psen_scan_v2/pull/189
- [x] https://github.com/PilzDE/psen_scan_v2/pull/186
- [x] https://github.com/PilzDE/psen_scan_v2/pull/188

**Add rosdoc_lite into CI**

The file `.rosdoc_lite_ignore` is needed since some warnings would need to be fixed upstream (e.g. https://github.com/ros-infrastructure/rosdoc_lite/pull/100) , yet it seems there is not much activity going on in https://github.com/ros-infrastructure/rosdoc_lite

Eventually this could be added to https://github.com/PilzDE/industrial_ci_addons or even https://github.com/ros-industrial/industrial_ci

### Things to add, update or check by the maintainers before this PR can be merged.

- [x] ~Public api function documentation~
- [x] ~Architecture documentation reflects actual code structure~
- [x] ~[Tutorials](https://wiki.ros.org/pilz_robots/Tutorials/)~
- [x] ~Overview on [ROS wiki](https://wiki.ros.org/pilz_robots)~
- [x] ~Package Readme ([example pilz_robots](https://github.com/PilzDE/pilz_robots/blob/melodic-devel/README.md))~
- [x] Good commit messages ([some tips](https://dev.to/jacobherrington/how-to-write-useful-commit-messages-my-commit-message-template-20n9))
- [x] ~CHANGELOG.rst updated~
- [x] Copyright headers
- [x] ~Examples~

### Review Checklist
- [x] ~Soft- and hardware architecture (diagrams and description)~
- [x] Test review (test plan and individual test cases)
- [x] Documentation describes purpose of file(s) and responsibilities
- [x] Code (coding rules, style guide)

### Release planning (please answer)
- [x] ~When is the new feature released?~
- [x] ~Which dependent packages do have to be released simultaneously?~

### CODE REVIEW

File: .github/workflows/ci_rosdoc_lite_check.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+jobs:
+  rosdoc_lite:
+    name: "rosdoc_lite check for psen_scan_v2"
+    runs-on: ubuntu-20.04
+    container: ros:noetic-ros-core
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install rosdoc-lite
+        run: sudo apt-get update && sudo apt-get install ros-noetic-rosdoc-lite graphviz -y
```
Comment (martiniil) [inline .github/workflows/ci_rosdoc_lite_check.yml:13]: I think `graphviz` can be removed?

**Review by martiniil — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_rosdoc_lite_check.yml (hunk 1 above)
Comment (martiniil) [reply .github/workflows/ci_rosdoc_lite_check.yml:13]: Apart from this it looks good.

Comment (martiniil) [inline .github/workflows/ci_rosdoc_lite_check.yml:13]:
```suggestion
        run: sudo apt-get update && sudo apt-get install ros-noetic-rosdoc-lite doxygen -y
```

Comment (agutenkunst) [reply .github/workflows/ci_rosdoc_lite_check.yml:13]: Should not be needed https://github.com/ros-infrastructure/rosdoc_lite/blob/981c151d22c567475bc0834729c9f282985c18b1/package.xml#L25

**Pull request merged**